### PR TITLE
Add component unit tests

### PR DESCRIPTION
**Pull request by chentong7**

### CODE REVIEW

File: packages/tools/client-debugger/client-debugger/src/test/ClientUtilities.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+		audience: services.audience,
+	};
+}
+
+/**
+ * Loads an existing Container for the given ID.
+ *
+ * @throws If no container exists with the specified ID, or if loading / connecting fails for any reason.
+ */
+export async function loadExistingFluidContainer(
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger/src/test/ClientUtilities.ts:80]: I don't think tests in this package will need this code path. This is really only for our sandbox app in the case where the user navigates to the URL with the container ID already specified. Our tests will likely always need to generate new containers.

File: packages/tools/client-debugger/client-debugger/package.json [hunk 2]
```diff
@@ -92,11 +92,18 @@
 		"eslint-plugin-chai-expect": "~3.0.0",
 		"mocha": "^10.0.0",
 		"nyc": "^15.0.0",
+		"jest": "^26.6.3",
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger/package.json:95]: This package shouldn't need jest. We're using `mocha` for testing in this package, since we don't do anything with UI / the browser. We should probably revert these changes.

Comment (chentong7) [reply packages/tools/client-debugger/client-debugger/package.json:95]: I used `jest`'s not null to verify when new container generated, debugger size will change and audience is not null. I didn't find `mocha` has not null/empty verify.

File: packages/tools/client-debugger/client-debugger/src/test/ClientUtilities.ts (hunk 1 above)
Comment (chentong7) [reply packages/tools/client-debugger/client-debugger/src/test/ClientUtilities.ts:80]: Make sense! Removing this function.

File: packages/tools/client-debugger/client-debugger/package.json (hunk 2 above)
Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger/package.json:95]: The `chai` library has the `expect` library we use when we're using mocha. It should have functionality like `expect(foo).to.not.be.null`. It's syntactically different from `jest`'s expect, but it should do the same thing.

**Comment by Josmithr** [comment]
You'll want to retarget this PR to point to `main` now that the initial library has been checked in.

File: packages/tools/client-debugger/client-debugger/package.json (hunk 2 above)
Comment (chentong7) [reply packages/tools/client-debugger/client-debugger/package.json:95]: I see! Modifying it. Thx!

**Comment by chentong7** [comment]
Merge this change into https://github.com/microsoft/FluidFramework/pull/13366. Closing this one...
